elytron/EAP7-1856-Bearer-Token-Authorization-Timeout.adoc

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+  - elytron
+---
+= [Preview] Bearer token timeout configurability will be added to WildFly's Elytron subsystem.
+:author:            Rebecca Searls
+:email:             rsearls@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:
+
+== Overview
+
+Bearer Token Authorization is the process of authorizing HTTP requests based on
+the existence and validity of a bearer token.  The token carries within it
+an expiration timestamp.  The two parameters being added, connection-timeout
+and read-timeout are placed on the URL used in retrieving the public key.
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-5691[WFCORE-5691]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1856[EAP7-1856]
+* https://issues.redhat.com/browse/ELY-2189[ELY-2189]
+* https://issues.redhat.com/browse/EAPSUP-640[EAPSUP-640]
```

Comment:
We can remove the reference to the EAPSUP I think.

Reply:
done